### DISCUSSION TITLE
Viikon 7 päivitys

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#432] [Tiedotteet]
Viikon 7 päivitys tehdään 14.2.2023 aamulla ja se pitää sisällään lokitusmuutoksia:

## Lokitusmuutokset

Lokitus on ollut tarpeettoman runsasta ja sen arvellaan nyt aiheuttavan satunnaisia joidenkin sekuntien viiveitä Kohan toiminnassa. Lokituksen määrän vähentäminen tulee näkymään käyttäjille viiveiden määrän vähenemisenä.

### SIPoHTTPS- rajapinnan lokitusta vähennetään

SIPoHTTPS-rajapinnan eli lainaus- ja palautusautomaattien sekä omatoimen ovikoneiden käyttämän rajapinnan lokitusta vähennetään.

### Plackin access-lokitus lopetetaan

Tähän asti kaikki palvelimelle tulleet sivupyynnöt on lokitettu. Vuositasolla tämä on tarkoittanut n. 800 miljoonaa - yhtä miljardia lokiriviä. Nämä lokirivit ovat periaatteessa nykyään turhia, koska pyyntöjen alkuperä on nykyisessä konesalissamme aina oma palomuurimme. Näiden pyyntöjen lokittaminen lopetetaan. Pyyntöjen alkuperä on edelleen seurattavissa palomuurilla.

**Comment by CodoDerDritte** [comment]
Samalla lopetettiin myös Apachen access-lokitus, joka niinikään keräsi hurjan määrän turhaa dataa.
